frontend/web/src/app/stl-viewer/classes/LoaderSelector.ts

```diff
@@ -0,0 +1,39 @@
+/**
+	@class LoaderSelector
+ */
+export class LoaderSelector {
+    loaderTable = {}
+
+    /**
+        Register a scene loader for a specific file format, using the file extesion name for lookup.
+        @param {String} fileExtName extension name for the specific file format.
+        @param {Function} loaderCtor constructor of the loader class.
+        */
+    registerLoader(fileExtName, loaderCtor) {
+        if((typeof loaderCtor) == 'function') {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') == should be === [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1440)